lib/gds_api/content_api.rb

```diff
@@ -52,13 +52,12 @@ def child_tags(tag_type, parent_tag, options={})
   end
 
   def tag(tag, tag_type=nil)
-    url = "#{base_url}/tags"
-
-    if tag_type
-      url << "/#{CGI.escape(tag_type)}"
+    if tag_type.nil?
```

Comment:
I felt that it wasn't worth the extra back-and-forth of emitting a deprecation warning, especially if I'm correct about the only client being Whitehall, which I'm migrating.